### DISCUSSION TITLE
Add Japanese translation to "Typing issue for Mozc"

### DISCUSSION THREAD

**Discussion by utuhiro78** [google/mozc#1126] [General]
[Typing issue for Mozc](https://docs.google.com/forms/d/e/1FAIpQLSfQ7UVkk9HxVQOQ1F2O_E3L5psvkkk-zPKJxJCbyAg9dZk8Yw/viewform)
Please add Japanese translation after English messages.
I think some Japanese users don't understand English well.
They will close the page silently.

e.g.
Category / カテゴリー
category of typing issue / タイピング問題のカテゴリー
1. OOV: Out-of-vocabulary / 語彙がない
e.g. "夕日" is not in the candidate list at all / "夕日" が候補リストにまったくない.

**Comment by faketanaka** [comment]
これは重要な問題です。日本語IMEなのですから、日本語話者を考慮する必要があります。

**Comment by hiroyuki-komatsu** [comment]
Since the form is not a part of the production but it is a part of the project page, it's not designed for the end-users.

That said, I'm fine to append Japanese messages too if you are willing to draft the messages in Japanese :-)

Thanks,
